### PR TITLE
ADD: Basic Doc for Theme Manager

### LINKED ISSUE #251: Missing Link for Menu Doc
The main documentation is missing a link to the menu documentation.

### PR DESCRIPTION
**Pull request by Mimonsi**
The numbering in the tutorial doesn't work... I don't know why. Please fix if you know why 👍 

closes #251

### CODE REVIEW

**Comment by iTitus** [comment]
If you count just write 1. 1. 1. 1. 1.

**Comment by Thabb** [comment]
Isn't that for .md?
For adoc it would be:
. First
. Second
. Third

Or if you want to number captions use
:numbered:
above the captions.

File: doc/developer_guide/Themes/index.adoc [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+When changing the active link:Theme.adoc[Theme], the link:ThemeManager.adoc[ThemeManager] will call `Refresh()` on all active link:ThemeTarget.adoc[ThemeTarget]s. Inactive ones will `Refresh()` themselves when enabled.
+
+=== How to add Themes to a new scene
+
+To add the Theme functionality to a new scene, do the following:
+
+. Create a new instante of link:Theme.adoc[Theme] and create the needed `Theme components` with colors.
+For example: `UI_background`, `button_normal`, `menu_button_highlighted`
+
+image::images/Theme.png[Theme scriptable view]
```
Comment (iTitus) [inline doc/developer_guide/Themes/index.adoc:45]:
```suggestion
image:images/Theme.png[Theme scriptable view]
```

File: doc/developer_guide/Themes/index.adoc [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+. Attach the link:ThemeManager.adoc[ThemeManager] to any objects, preferable the HUD. Make sure to insert an active link:Theme.adoc[Theme] and (optionally) more scenes in the `Selectable link:Theme.adoc[Theme]s` Array.
+ 
+image::images/ThemeManager.png[ThemeManager attached to HUD-GameObject in Editor]
```
Comment (iTitus) [inline doc/developer_guide/Themes/index.adoc]:
```suggestion
image:images/ThemeManager.png[ThemeManager attached to HUD-GameObject in Editor]
```

File: doc/developer_guide/Themes/index.adoc [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+. Attach the link:ThemeTarget.adoc[ThemeTarget] to every UI-Element with link:Theme.adoc[Theme] functionality. There are many subclasses of `link:ThemeTarget.adoc[ThemeTarget]` to be used - for example `ImageTarget` for Images, `ButtonTarget` for Buttons and Toggles.
+
+image::images/ImageTarget.png[ImageTarget with color preview]
```
Comment (iTitus) [inline doc/developer_guide/Themes/index.adoc]:
```suggestion
image:images/ImageTarget.png[ImageTarget with color preview]
```

**Review by iTitus — CHANGES_REQUESTED** [review]
try this

**Review by iTitus — APPROVED** [review]

**Pull request merged**